src/test/java/org/folio/rest/GeneratePasswordRestLinkTest.java

```diff
@@ -92,6 +92,7 @@ public static void setUp(TestContext context) {
       .setContentType(ContentType.JSON)
       .setBaseUri("http://localhost:" + port)
       .addHeader(RestVerticle.OKAPI_HEADER_TENANT, TENANT)
+      .addHeader(RestVerticle.OKAPI_HEADER_TOKEN, "test_token")
```

Comment:
Why is this needed?

Reply:
I got NPE when we try to copy received headers in `org.folio.service.PasswordResetLinkServiceImpl#sendPasswordRestLink` endpoint  when calling `org.folio.rest.util.OkapiConnectionParams#buildHeaders`.

Probably `io.vertx.core.http.CaseInsensitiveHeaders` implementation changed and it is not allow nulls anymore

Reply:
> I got NPE when we try to copy received headers in `org.folio.service.PasswordResetLinkServiceImpl#sendPasswordRestLink` endpoint when calling `org.folio.rest.util.OkapiConnectionParams#buildHeaders`.

Thanks for that explanation. Please could a comment be added to explain that in the code.

Also, if not too much work, please could a more representative token be used (like in mod-circulation)

> Probably `io.vertx.core.http.CaseInsensitiveHeaders` implementation changed and it is not allow nulls anymore

Hasn't that been replaced with `caseInsensitiveMultiMap` everywhere?

Reply:
Yes, it has. But this method internally uses that deprecated class.

Reply:
> Also, if not too much work, please could a more representative token be used (like in mod-circulation).

Done.